upgrade_tests/test_scenarios/test_role.py

```diff
@@ -15,10 +15,11 @@
 :Upstream: No
 """
 
-from upgrade_tests import pre_upgrade, post_upgrade
+from unittest2.case import TestCase
```

Comment:
Btw, Why unittest, we are not going to follow unittest approach? Only pytest ?

Reply:
will update

Reply:
@jyejare I looked into previous automated tests and they all use unittest , do you need me to update all the scenario tests to use pytest?

Reply:
Yeah, Please do.

Reply:
Sorry #sanket for my comment. I see we are using Unittest assert methods in scenario tests.
 Please revert, Sorry again